.circleci/config.yml

```diff
@@ -0,0 +1,94 @@
+version: 2
+defaults: &defaults
+  docker:
+    - image: circleci/node:8
+  working_directory: ~/nodejs
+
+jobs:
+  install:
+    <<: *defaults
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+          - v1-yarn-cache-{{ checksum "yarn.lock" }}
+          - v1-yarn-cache-
+      - run:
+          name: Install greenkeeper-lockfile-update
+          command: yarn global add greenkeeper-lockfile@1 --prefix ~/.yarn
+      - run:
+          name: Install dependencies
+          command: yarn install --pure-lockfile
+      - save_cache:
+          key: v1-yarn-cache-{{ checksum "yarn.lock" }}
+          paths:
+            - node_modules
+      - run:
+          name: Update greenkeeper-lockfile
+          command: ~/.yarn/bin/greenkeeper-lockfile-update
+      - run:
+          name: Upload greenkeeper-lockfile
+          command: ~/.yarn/bin/greenkeeper-lockfile-upload
+  lint:
+    <<: *defaults
+      - run:
+          name: Run linters
+          command: yarn lint
+  test_unit:
+    <<: *defaults
+      - run:
+          name: Run tests
+          command: yarn test:coverage:ci --maxWorkers=10
+  test_integration:
+    <<: *defaults
+      - run:
+          name: Run integration tests
+          command: yarn test:integration
+  prepare_release:
+    <<: *defaults
+      - run:
+          name: Run prepare release
+          command: ./.circleci/prepare-release-packages.sh
+  perform_release:
+    <<: *defaults
+      - run:
+          name: Run perform release
+          command: ./.circleci/perform-release-packages.sh
+
+workflows:
+  version: 2
+  linting:
```

Comment:
I felt like using progressive tense (hope that's correct :smile:) to separate naming of job from workflows. I often got lost in other configs and felt this might help. Maybe it makes sense. I saw the job as a blueprint to be instantiated/run in a job.

Reply:
Have you tried this already? I fail to see how you can have dependencies between different workflows. As far as I know you can do that within the same workflow, not across different ones.

What is the idea of doing it like this? 🤔